.travis.yml

```diff
@@ -50,12 +50,15 @@ before_install:
     fi
 script:
   - rustc --version
-  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.4" ]; then
+  - if [ "$TRAVIS_RUST_VERSION" == "nightly" ] && [ "$GTK" == "3.14" ]; then
     make regen_check;
     fi
   - ./check_init_asserts
-  - cargo doc --features "dox"
-  - cargo test --features "$FEATURES"
+  - cargo doc --features "dox,embed-lgpl-docs"
+  - cargo test --features "$FEATURES,embed-lgpl-docs"
```

Comment:
I still slightly worry about updating all *.rs on second `embed-lgpl-docs` but 👍 anyway

Reply:
That's how it is in the other `.travis.yml`. See https://github.com/gtk-rs/pango/pull/136#discussion_r255456569